recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2.2-rc1" %}
+
+package:
+  name: triqs
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/triqs/archive/{{ version }}.tar.gz
+  sha256: cf4ce7ea7ba873a5bbb3fd53aa544106dc7e76970d5962ba19e27c240b5b1641
+
+build:
+  number: 0
+  skip: True  # [win or py >2]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - py <3
+    - pip
+  host:
+    - boost
+    - fftw
+    - gmp
+    - hdf5
+    - h5py
+    - mako
+    - matplotlib
+    - mpi4py
+    - numpy
+    - openmpi
+    - scipy
+  run:
+    - boost
+    - clangxx >=8
+    - fftw
+    - gmp
+    - hdf5
+    - h5py
+    - matplotlib
+    - mpi4py
+    - numpy
+    - openmpi
+    - py <3
+    - pip
+    - scipy
+
+test:
+  imports:
+    - pytriqs
+
+about:
+  home: https://triqs.github.io
+  license: GPLv3
```

Comment:
Also, we were wondering how to specify that our project is using two different licenses for different components (GPLv3 and Apache2)

Reply:
I don't think that you can do that. It seems really strange to me to distribute a single *package* under two different licenses that apply to different parts of it. If you really want bits of it to be Apache 2, I would recommend to split this into two different feedstocks. Otherwise, Apache 2 licensed code can be distributed under GPLv3 so the current value looks good to me.

Reply:
@saraedum Thanks for clarifying! Let's keep it as GPLv3